services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/Incident.java

```diff
@@ -0,0 +1,157 @@
+package com.mapbox.api.directions.v5.models;
+
+import androidx.annotation.NonNull;
+import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.mapbox.api.directions.v5.DirectionsAdapterFactory;
+
+/**
+ * Object representing an incident along the intersection.
+ *
+ */
+@AutoValue
+public abstract class Incident extends DirectionsJsonObject {
+
+  /**
+   * Incident type.
+   *
+   * @return incident type
+   */
+  @NonNull
```

Comment:
Capturing from https://github.com/mapbox/mapbox-java/pull/1177#issuecomment-689702542, this and all other fields (besides `id`) need to be nullable.

Reply:
fixed